extra/exports

```diff
@@ -151,6 +151,7 @@ box_txn_rollback_to_savepoint
 box_txn_savepoint
 box_txn_set_isolation
 box_txn_set_timeout
+box_txn_make_sync
```

Comment:
Please, keep the file sorted. It is explained in the beginning of it.

Reply:
Fixed.